action.yml

```diff
@@ -12,6 +12,10 @@ runs:
   # Requires that the pull request's merge commit is checked out
   # to the current working directory.
   steps:
+    - uses: zendesk/setup-ruby@v1.192.0
+      with:
+        bundler-cache: true
+
```

Comment:
Nope, we shouldn’t need to install all of a project’s gems before running our little script. I’ll open a PR to remove these two lines.